scripts/kubectl-kubestellar-ensure-location

```diff
@@ -108,13 +108,25 @@ fi
 
 set -e
 
+echo "--- current directory is $PWD"
+cwsi=$(kubectl ws "${kubectl_flags[@]}" .)
+cwsi=${cwsi#*'"'}; cwsi=${cwsi%'"'*}
+echo "current ws is $cwsi"
+echo "imw is $imw"
 if [ "$imw" != "." ]
 then kubectl ws "${kubectl_flags[@]}" "$imw"
+     cwsi=$imw
+fi
+echo "current cwsi is $cwsi"
+if ! kubectl get apibinding "edge.kcp.io" &> /dev/null; then
+   kubectl kcp bind apiexport root:espw:edge.kcp.io
```

Comment:
FYI: In the other scripts that depend on the ESPW, the workspace pathname of the ESPW is an optional command-line flag.